packages/circuits/scripts/downloadPtau.mjs

```diff
@@ -9,13 +9,15 @@ import { ptauName } from './circuits.mjs'
 const __dirname = path.dirname(url.fileURLToPath(import.meta.url))
 
 const outDir = path.join(__dirname, '../zksnarkBuild')
+const buildDir = path.join(__dirname, '../zksnarkBuild.tmp')
```

Comment:
This file should be fine. It already operates atomically, if a build is cancelled mid-download it's safe. If you want to move the in progress download to a proper tmpdir that's fine though.

Reply:
`buildDir` now is `os.tmpdir()`